sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/helpers/UniqueIdHelper.java

```diff
@@ -5,11 +5,12 @@
 import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
 
 import java.net.HttpURLConnection;
+import java.util.Objects;
 import java.util.function.Function;
 
 public class UniqueIdHelper {
     public static String getUniqueModelId(String baseName, DigitalTwinsAsyncClient client, Function<Integer, String> randomIntegerStringGenerator) {
-        return getUniqueId(baseName, (modelId -> client.getModel(modelId).block().getId()), randomIntegerStringGenerator);
+        return getUniqueId(baseName, (modelId -> Objects.requireNonNull(client.getModel(modelId).block()).getId()), randomIntegerStringGenerator);
```

Comment:
don't we need it everywhere in this file?

Reply:
the required non-null check? I added it here because intellij recommended that I do; we should go through our code to see if it makes sense to add it  elsewhere as well.